pkg/controller/virtualmachineimage/importer_pod.go

```diff
@@ -32,15 +28,21 @@ const (
 	ImporterImageSize = "IMPORTER_IMAGE_SIZE"
 	// InsecureTLSVar provides a constant to capture our env variable "INSECURE_TLS"
 	InsecureTLSVar = "INSECURE_TLS"
-	// SourceHTTP is the source type HTTP, if unspecified or invalid, it defaults to SourceHTTP
+	// SourceHTTP is the source type HTTP
 	SourceHTTP = "http"
+	// SourceHostPath is the source type volume
+	SourceHostPath = "hostPath"
 	// ImageContentType is the content-type of the imported file
 	ImageContentType = "kubevirt"
 	// ImportPodImage and ImportPodVerbose should be modified to get value from vmi env
 	// ImportPodImage indicates image name of the import pod
 	ImportPodImage = "kubevirt/cdi-importer:v1.13.0"
 	// ImportPodVerbose indicates log level of the import pod
 	ImportPodVerbose = "1"
+	// SourceVolumeName provides a const to use for creating source volume in pod specs
```

Comment:
SourceVolumeName is used for creating source volume in pod specs 
is more clear.